Qt5/5.13.1-GCCcore-8.3.0 depends on JasPer.

I'm trying to build Qt5/5.13.1-GCCcore-8.3.0 on RHEL8 and it fails on:

```
qjp2handler.cpp:47:10: fatal error: jasper/jasper.h: No such file or directory
 #include <jasper/jasper.h>
          ^~~~~~~~~~~~~~~~~
```

JasPer is not set as a dependency for Qt5/5.13.1-GCCcore-8.3.0 and there is no JasPer/2.0.14-GCCcore-8.3.0 available. I'm mostly wondering if this is a RHEL8 issue since this hasn't been seen before? There is a jasper-devel package available in the OS, but there are no osdeps either in the package.

For what its worth, JasPer/2.0.14 builds fine with GCCcore 8.3.0 and Qt5/5.13.1-GCCcore-8.3.0 builds fine once JasPer/2.0.14-GCCcore-8.3.0 is installed and added as a dependency.

Hm, I'm not sure what this CI error stems from.

```
/opt/hostedtoolcache/Python/3.7.5/x64/lib/python3.7/site-packages/pep8.py:110: FutureWarning: Possible nested set at position 1
  EXTRANEOUS_WHITESPACE_REGEX = re.compile(r'[[({] | []}),;:]')
.
======================================================================
ERROR: test_dep_graph (test.easyconfigs.easyconfigs.EasyConfigTest)
Unit test that builds a full dependency graph.
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/runner/work/easybuild-easyconfigs/easybuild-easyconfigs/test/easyconfigs/easyconfigs.py", line 132, in test_dep_graph
    dep_graph(fn, EasyConfigTest.ordered_specs)
  File "/opt/hostedtoolcache/Python/3.7.5/x64/lib/python3.7/site-packages/easybuild/framework/easyconfig/tools.py", line 210, in dep_graph
    dgr.add_edge((spec['module'], dep))
  File "/opt/hostedtoolcache/Python/3.7.5/x64/lib/python3.7/site-packages/pygraph/classes/digraph.py", line 169, in add_edge
    raise AdditionError("Edge (%s, %s) already in digraph" % (u, v))
pygraph.classes.exceptions.AdditionError: Edge (Qt5/5.13.1-GCCcore-8.3.0, JasPer/2.0.14-GCCcore-8.3.0) already in digraph
----------------------------------------------------------------------
Ran 8674 tests in 443.516s
FAILED (errors=1)
ERROR: Not all tests were successful.
Found 8665 easyconfigs...

List of changed easyconfig files in this PR: JasPer-2.0.14-GCCcore-8.3.0.eb
Qt5-5.13.1-GCCcore-8.3.0.eb
##[error]Process completed with exit code 2.
```

--- a/easybuild/easyconfigs/j/JasPer/JasPer-2.0.14-GCCcore-8.3.0.eb
+++ b/easybuild/easyconfigs/j/JasPer/JasPer-2.0.14-GCCcore-8.3.0.eb
@@ -1,0 +1,35 @@
+easyblock = 'CMakeMake'
+
+name = 'JasPer'
+version = '2.0.14'
+
+homepage = 'https://www.ece.uvic.ca/~frodo/jasper/'
+
+description = """
+ The JasPer Project is an open-source initiative to provide a free
+ software-based reference implementation of the codec specified in
+ the JPEG-2000 Part-1 standard.
+"""
+
+toolchain = {'name': 'GCCcore', 'version': '8.3.0'}
+toolchainopts = {'pic': True}
+
+source_urls = ['https://www.ece.uvic.ca/~frodo/jasper/software/']
+sources = [SOURCELOWER_TAR_GZ]
+checksums = ['2a1f61e55afe8b4ce8115e1508c5d7cb314d56dfcc2dd323f90c072f88ccf57b']
+
+builddependencies = [
+    ('binutils', '2.32'),
+    ('CMake', '3.15.3'),
+]
+
+separate_build_dir = True
+
+configopts = '-DJAS_ENABLE_DOC=OFF '
+
+sanity_check_paths = {
+    'files': ['bin/jasper', ('lib/libjasper.%s' % SHLIB_EXT, 'lib64/libjasper.%s' % SHLIB_EXT)],
+    'dirs': ['include'],
+}
+
+moduleclass = 'vis'

--- a/easybuild/easyconfigs/q/Qt5/Qt5-5.13.1-GCCcore-8.3.0.eb
+++ b/easybuild/easyconfigs/q/Qt5/Qt5-5.13.1-GCCcore-8.3.0.eb
@@ -50,6 +50,7 @@ dependencies = [
     ('libjpeg-turbo', '2.0.3'),
     ('NSS', '3.45'),
     ('snappy', '1.1.7'),
+    ('JasPer', '2.0.14'),
 ]
 
 # qtgamepad needs recent kernel/libevdev (fails on RHEL 6.x)

